api/pom.xml

```diff
@@ -12,60 +12,6 @@
     <name>Open Concept Lab Module API</name>
     <description>API project for OpenConceptLab</description>
 
-    <dependencies>
-        <!-- Begin OpenMRS core -->
-
-        <dependency>
-            <groupId>org.openmrs.api</groupId>
-            <artifactId>openmrs-api</artifactId>
-            <type>jar</type>
-        </dependency>
-
-        <dependency>
-            <groupId>org.openmrs.api</groupId>
-            <artifactId>openmrs-api</artifactId>
-            <type>test-jar</type>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.openmrs.web</groupId>
-            <artifactId>openmrs-web</artifactId>
-            <type>test-jar</type>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.openmrs.test</groupId>
-            <artifactId>openmrs-test</artifactId>
-            <type>pom</type>
-            <scope>test</scope>
-        </dependency>
-
-        <!-- In 1.9, this is included in openmrs-web
-             here we treat it as already being provided -->
-        <dependency>
-            <groupId>org.codehaus.jackson</groupId>
-            <artifactId>jackson-mapper-asl</artifactId>
-            <version>1.5.0</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <!-- End OpenMRS core -->
-
-        <dependency>
-            <groupId>commons-httpclient</groupId>
-            <artifactId>commons-httpclient</artifactId>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-compress</artifactId>
-            <version>1.21</version>
-        </dependency>
-
-    </dependencies>
-
```

Comment:
When I added the 2.2 profile, it took some trial and error to get things to pass.  To make things a little easier to figure out, I moved all of the shared api/omod dependencies into the top-level pom, and all of the omod-specific dependencies into the omod pom, and got rid of dependencyManagement.